static/robots.txt

```diff
@@ -0,0 +1,17 @@
+# exclude popular wikipedia snapshots published without link rel=canonical
+# context: https://github.com/ipfs/website/pull/334
+# en.wikipedia.org @ 2017-05-11
+# simple.wikipedia.org @ ?
+# ku.wikipedia.org @ 2017-05-10
+# tr.wikipedia.org @ 2017-04-30
+User-agent: *
+Disallow: /ipfs/QmXoypizjW3WknFiJnKLwHCnL72vedxjQkDDP1mXWo6uco/
+Disallow: /ipfs/QmU2TE8ym8rBAMwzaanZ3SbVtBJY4R7oX2uN1fca4JAnrD/
+Disallow: /ipfs/QmWY4KZXKTuspGSwYVDNbNLLZcmSiQ63Mdmz7eRd4KzBbb/
+Disallow: /ipfs/QmT5NvUtoM5nWFfrQdVrFtvGfKFmG7AHE8P34isapyhCxX/
+
+# better safe than sorry
+User-agent: *
+Disallow: /harming/humans
```

Comment:
They will work this around with 0th law. ;p

Reply:
:grimacing: PR welcome :upside_down_face: